bin/cli

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+require('../index')(process.argv[2])
+  .then(result => {
+    if (!result.isUpToDate) {
+      result.errors.forEach(e => console.error(e))
+      process.exit(1)
+    }
+  })
+  .catch(e => {
+    console.error(e)
+    process.exit(1)
+  })
```

Comment:
When I run this on my local copy of `otp-react-redux` with the `alert+map-fixes` branch checked out, I get this error `fatal: no upstream configured for branch \'alert+map-fixes\'` even though it is up to date with the remote repo.

Reply:
Update: I hadn't done `git push -u origin alert+map-fixes` on my local repo which was causing rev-parse to fail. Perhaps the git commands should be surrounded in a try/catch with an error message added if it fails to tell the user to run `git push -u origin $branch` to set the upstream?

Reply:
See #2.